doc/adrs/143_handle_documents_upload_with_other_formats_than_json_and_xmi.adoc

```diff
@@ -0,0 +1,34 @@
+= [ADR-143] Handle documents upload with other formats than JSON and XMI
+
+== Context
+
+Sirius Web doesn't provide a way for specifiers to handle documents uploads with other formats than JSON and XMI.
+
+== Decision
+
+=== Backend
+
+A new `IResourceLoaderService.java` interface will be introduced:
+
+[source,java]
+----
+public interface IResourceLoaderService {
+    boolean canHandle(InputStream inputStream, URI resourceURI, ResourceSet resourceSet);
+    Resource getResource(InputStream inputStream, URI resourceURI, ResourceSet resourceSet);
```

Comment:
Small detail but I'm pretty sure that it should return an Optional<Resource>. While in canHandle, you can detect that the input stream has the proper type (i.e. start with `{` for json), during the loading itself there can be a syntax error crashing the deserialization process with an `IOException`.

I'll update that detail and merge the two ADRs